src/include/network/lan/address.rb

```diff
@@ -393,7 +393,7 @@ def initialize_network_lan_address(include_target)
       Ops.set(
         @widget_descr_local,
         "HWDIALOG",
-        Ops.get(@widget_descr_hardware, "HWDIALOG", {})
+        Ops.get(widget_descr_hardware, "HWDIALOG", {})
```

Comment:
above is `@widget_descr_local` I found quite confusing to have such mixture. Is it at least covered by some testsuite?